src/conventional/ionosonde_ascii2ioda.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import time
+from datetime import datetime
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import iso8601_string, epoch
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {'criticalFrequency': ['criticalFrequency', "float", 'm s-1'],
+           'criticalFrequencyConfidence': ['criticalFrequencyConfidence', "float", 'fractional percent'],
+           'height': ['height', "float", "m"],
+           'electronDensity': ['electronDensity', "float", 'number / m^3'],
+           'electronDensityConfidence': ['electronDensityConfidence', "float", 'number / m^3']}
```

Comment:
I looked at the converted files and found electronDensityConfidence is under all groups (ObsValue, ObsErrors...) as a separate variable. Is this correct? Should it be ObsErrors for electronDensity?

```
group: ObsError {
  variables:
  	float electronDensity(Location) ;
  		electronDensity:_FillValue = 9.96921e+36f ;
  		string electronDensity:units = "number / m^3" ;
  		string electronDensity:coordinates = "longitude latitude" ;
  	float electronDensityConfidence(Location) ;
  		electronDensityConfidence:_FillValue = 9.96921e+36f ;
  } // group ObsError

group: ObsValue {
  variables:
  	float electronDensity(Location) ;
  		electronDensity:_FillValue = 9.96921e+36f ;
  		string electronDensity:units = "number / m^3" ;
  		string electronDensity:coordinates = "longitude latitude" ;
  	float electronDensityConfidence(Location) ;
  		electronDensityConfidence:_FillValue = 9.96921e+36f ;
```